### DISCUSSION TITLE
Feature request: Different `title-delim`s for figures and tables

### DISCUSSION THREAD

**Discussion by mine-cetinkaya-rundel** [quarto-dev/quarto-cli#4365] [Feature Requests]
Currently `title-delim` can be defined commonly for figures and tables (https://quarto.org/docs/authoring/cross-references.html#titles). It would be nice to be able to define `fig-title-delim` and `tbl-title-delim` separately.

cc @zief0002

**Comment by mcanouil** [comment]
I opened an issue for this.
Closing this thread in favour of https://github.com/quarto-dev/quarto-cli/issues/6258
